src/fdb5/api/helpers/ArchiveCallback.h

```diff
@@ -20,8 +20,10 @@
 
 namespace fdb5 {
 
-using ArchiveCallback = std::function<void(const Key&, const FieldLocation&)>;
+using ArchiveCallback = std::function<void(const Key& key, const void* data, size_t length, const FieldLocation&)>;
+using FlushCallback = std::function<void()>;
 
-static const ArchiveCallback CALLBACK_NOOP = [](const Key&, const FieldLocation&) {};
+static const ArchiveCallback CALLBACK_NOOP = [](const Key& key, const void* data, size_t length, const FieldLocation&) {};
+static const FlushCallback CALLBACK_FLUSH_NOOP = []() {};
```

Comment:
This file is called ArchiveCallback.h - clearly wrong if FlushCallback is there.